### PR TITLE
Replace old OBJLoader with new version

### PR DESCRIPTION
**Pull request by justin-hackin**
From here https://github.com/mrdoob/three.js/blob/cf584a60bdfd24c42eaa81d484533364742bda44/examples/js/loaders/OBJLoader.js

Works as expected with latest npm package 'three' (R86)

### CODE REVIEW

**Comment by justin-hackin** [comment]
Still emits warning i.e.:
```
./~/babel-loader/lib!./~/vue-loader/lib/selector.js?type=script&index=0!./src/components/ShapeLoader.vue
12:20-35 "export 'OBJLoader' (imported as 'THREE') was not found in 'three'
```
but it seems to work just fine

File: source/index.js [hunk 1]
```diff
@@ -1,751 +1,403 @@
...
-
-          var multiMaterial = new THREE.MultiMaterial(createdMaterials);
-          mesh = (!isLine ? new THREE.Mesh(buffergeometry, multiMaterial) : new THREE.LineSegments(buffergeometry, multiMaterial));
-
-        } else {
-
-          mesh = (!isLine ? new THREE.Mesh(buffergeometry, createdMaterials[0]) : new THREE.LineSegments(buffergeometry, createdMaterials[0]));
         }
 
+        material.shading = object.material.smooth ? THREE.SmoothShading : THREE.FlatShading;
```
Comment (Winston-Guess) [inline source/index.js:391]: `material.shading` has been removed.  `material.flatShading = true` should be used instead.

**Review by Winston-Guess — CHANGES_REQUESTED** [review]

File: source/index.js (hunk 1 above)
Comment (sohamkamani) [reply source/index.js:391]: Thats true..perhaps that could be done in another PR?

Comment (Winston-Guess) [reply source/index.js:391]: @sohamkamani That makes sense. ~ I don't know the PR procedure outside of my work environment, new to the game 👍

Comment (danvk) [reply source/index.js:391]: I sent a PR with just this fix https://github.com/sohamkamani/three-object-loader/pull/15

Comment (sohamkamani) [reply source/index.js:391]: merged 👍  Sorry for the (very) late reply

**Pull request closed without merging**